### ISSUE TITLE
Feat: Change position of item of Action enum

### ISSUE THREAD

**Issue by magj2006** [ExocoreNetwork/exocore-contracts#104]
Considering that the items in the `Action` enum may change, I propose moving `Respond` and `AddToken` to the front of the `Action` enum. The main reason for this proposal is that the Solana client, upon receiving messages from the `ExocoreGateway`, executes logic based on the `action` in the message. If there are changes to the items in the `Action` enum, it may affect the values of these two items, forcing the Solana client to update accordingly to accommodate such changes.

expect: 
enum Action {
   Respond,
   REQUEST_ADD_WHITELIST_TOKEN,
   ...
}

**Comment by MaxMustermann2** [comment]
Is the idea that we `fix` the position of the `enum` items forever, with those two items in the front? Related: #26

**Comment by adu-web3** [comment]
I think the things that really matter is that we should agree on a stable version for `Action` and use github CI to check that existed actions can not be changed anymore and only appending new ones are allowed

**Comment by adu-web3** [comment]
```
enum Action {
    RESPOND,
    REQUEST_ADD_WHITELIST_TOKEN,
    REQUEST_DEPOSIT_LST,
    REQUEST_DEPOSIT_NST,
    REQUEST_WITHDRAW_LST,
    REQUEST_WITHDRAW_NST,
    REQUEST_SUBMIT_REWARD,
    REQUEST_CLAIM_REWARD,
    REQUEST_DELEGATE_TO,
    REQUEST_UNDELEGATE_FROM,
    REQUEST_DEPOSIT_THEN_DELEGATE_TO,
    REQUEST_MARK_BOOTSTRAP,
    REQUEST_ASSOCIATE_OPERATOR,
    REQUEST_DISSOCIATE_OPERATOR
}
```

this could be a possibly stable version for Action

**Comment by magj2006** [comment]
> ```
> enum Action {
>     RESPOND,
>     REQUEST_ADD_WHITELIST_TOKEN,
>     REQUEST_DEPOSIT_LST,
>     REQUEST_DEPOSIT_NST,
>     REQUEST_WITHDRAW_LST,
>     REQUEST_WITHDRAW_NST,
>     REQUEST_SUBMIT_REWARD,
>     REQUEST_CLAIM_REWARD,
>     REQUEST_DELEGATE_TO,
>     REQUEST_UNDELEGATE_FROM,
>     REQUEST_DEPOSIT_THEN_DELEGATE_TO,
>     REQUEST_MARK_BOOTSTRAP,
>     REQUEST_ASSOCIATE_OPERATOR,
>     REQUEST_DISSOCIATE_OPERATOR
> }
> ```
> 
> this could be a possibly stable version for Action

Yeah, stable it like this.  and we can only add new item, not allow to delete.
